docs/Wallet.md

```diff
@@ -25,70 +19,96 @@ The wallet contains following features:
+:::danger
+Using the hot wallet feature comes with its own security implications, please be sure to read and understand them over at the [Hot Wallet documentation](HotWallet.md)
+:::
 
-### Receive
-
-Receive tab generates an unused address which can be used to receive payments. The same can be achieved by generating an invoice (Invoices > Create new invoice).
+#### Advanced Settings
 
-### Advanced Settings
+Certain wallet features are available for advanced users. Toggle the `Advanced Settings` within the `Send` tab to preview them.
 
-#### Don't create UTXO change
+##### Don't create UTXO change
 
-This option is available in the `Advanced mode` of the Send page.
+This option is available in the `Advanced mode` of the `Send` page.
 
-It is a privacy enhancing feature which is useful when you send to another of your own wallets or to an exchange. It makes sure that no change UTXO is created by **rounding up** the amount sent.
+It is a privacy enhancing feature which is useful when you're sending funds to another wallet of yours or to an exchange. It makes sure that no change UTXO is created by **rounding up** the amount sent.
 
-By default this feature is disabled, so if your wallet has a UTXO of `1.1 BTC` and you input an amount equals to `1.0 BTC`, the resulting transaction will have two outputs `0.1 BTC` of change, and `1.0 BTC` to your destination.
+By default this feature is disabled, so if your wallet has an UTXO of `1.1 BTC` and you input an amount equals to `1.0 BTC`, the resulting transaction will have two outputs `0.1 BTC` of change, and `1.0 BTC` to your destination.
 
 Blockchain analysis will understand that those `0.1 BTC` of change belong to the same entity which controlled `1.1 BTC` before, and can track the future purchase you make under the same pattern.
 
 By enabling this feature, BTCPay Server wallet will round up the amount sent to `1.1 BTC` such that no change output is sent back to you.
 
 Warning: Despite the fact, in this example, that you entered `1.0` in the amount field, the amount that will really be sent to your destination will be `1.1 BTC`.
 
+##### RBF (Replace-by-fee)
+
+##### Coin Selection
+
+Coin selection is a an advanced privacy-enhancing feature that allows you to specifically select coins that you would like to spend when crafting a transaction. For example, paying with coins that are fresh from a coinjoin mix.
+
+To make the selection easier, coin-selection works natively with the wallet labels feature. This allows you to label any incoming funds for smoother UTXO management and spending.
+
+##### Other features
+
+###### Camera QR scan
+
+Scan option in wallet (camera icon) lets you use your device’s camera to scan an invoice when sending from the wallet. It auto-populates the invoice information so you don’t have to manually copy-paste address and amount.
```

Comment:
```suggestion
Scan option in wallet (camera icon in send screen) lets you use your device’s camera to scan a QR code containing an address or BIP21 payment link. It auto-populates the sending information so that you don’t have to manually copy-paste an address and amount.
```